pages/common/wrk.md

```diff
@@ -0,0 +1,16 @@
+# wrk
+
+> HTTP benchmarking tool.
+> Homepage: <https://github.com/wg/wrk>.
+
+- Run a benchmark for `30` seconds, using `12` threads, and keeping `400` HTTP connections open:
+
+`wrk -t{{12}} -c{{400}} -d{{30s}} {{http://127.0.0.1:8080/index.html}}`
+
+- Run a benchmark with a custom header:
+
+`wrk -t{{2}} -c{{5}} -d{{5s}} -H {{‘Host: example.com’}} {{http://example.com/index.html}}`
```

Comment:
Oops! 1 small thing more that I  didn't notice before, and you don't seem to have given permission for maintainers to make edits.

The smart quotes here probably need replacing with straight ones:

```suggestion
`wrk -t{{2}} -c{{5}} -d{{5s}} -H "{{Host: example.com}}" "{{http://example.com/index.html}}"`
```

I've also added quotes around the url (just in case it has an ampersand `&` in it). Maybe you could do that for the other examples too please?

Reply:
@Aracki I'd suggest applying quotes around **all** the URLs since those can easily mess with the shell if they contain special characters (like `&`) and are not quoted.

Reply:
Fixed!